data/card_catalogue/movement.json

```diff
@@ -4,7 +4,10 @@
     "upgrade": "Charge",
     "tier": "blue",
     "details": {
-      "rules": "Move up to 5 squares in any direction",
+      "rules": {
+        "active": "Move up to 5 metres in any direction.",
```

Comment:
<del>squares</del> <ins>metres</ins>
<del>tiles</del> <ins>metres</ins>
<del>blocks</del> <ins>metres</ins>
<del>units</del> <ins>metres</ins>
<del>feet</del> <ins>metres</ins> (converted)
<del>miles</del> <ins>kilometres</ins> (converted)
<del>gallons</del> <ins>litres</ins>


have tagged these sorts of commits with the description `standardise units`, ex. f6c1b45e29b0feeb8800712798936eb05bba8671
I'm indifferent so if you prefer tiles/squares at least it'll be nice and easy to <kbd>ctrl</kbd> + <kbd>h</kbd>